### PR TITLE
prepublish addition. remove extra build from CI

### PR DESCRIPTION
**Pull request by braddialpad**
no longer need npm run build in CI as we do it as part of prepublish.

added npm run svg to prepublishOnly. npm run svg does NOT run when you execute npm run build. This was probably a big point of confusion in the manual release last night.

removed svg from setup:ci since we now run it in prepublishOnly

### CODE REVIEW

**Comment by dialpad-drew** [comment]
Projects won't successfully build without the SVGs, right? If that's the case, does that suggest not running `gulp svg` at setup could mean new projects won't successfully build?

I'm thinking about this from the perspective of a product engineer who's cloned the project and set it up for the first time so they can contribute. Does this work for them?

**Comment by braddialpad** [comment]
> Projects won't successfully build without the SVGs, right? If that's the case, does that suggest not running `gulp svg` at setup could mean new projects won't successfully build?
> 
> I'm thinking about this from the perspective of a product engineer who's cloned the project and set it up for the first time so they can contribute. Does this work for them?

there's a npm run setup command that is meant to be run when you initially set up the project. I still don't really like this because it goes outside the standard process most developers are used to which is "npm install" and then "npm run build" or "npm run watch". Not to mention you have to KNOW to run npm run setup again whenever there's any changes to SVGs.

Personally I think it may be best to include the svg as part of the build. Guessing this was set up as an independent script originally for performance reasons. I'll do some build time comparisons so we can make a decision.

**Comment by dialpad-drew** [comment]
> > Projects won't successfully build without the SVGs, right? If that's the case, does that suggest not running `gulp svg` at setup could mean new projects won't successfully build?
> > I'm thinking about this from the perspective of a product engineer who's cloned the project and set it up for the first time so they can contribute. Does this work for them?
> 
> there's a npm run setup command that is meant to be run when you initially set up the project. I still don't really like this because it goes outside the standard process most developers are used to which is "npm install" and then "npm run build" or "npm run watch". Not to mention you have to KNOW to run npm run setup again whenever there's any changes to SVGs.
> 
> Personally I think it may be best to include the svg as part of the build. Guessing this was set up as an independent script originally for performance reasons. I'll do some build time comparisons so we can make a decision.

Dang, I just noticed you removed the SVG task from `setup:ci`, not `setup` entirely. In that case, I think my question is actually moot because the situation I described shouldn't occur if someone is following the install directions. 🤦

**Comment by braddialpad** [comment]
> Dang, I just noticed you removed the SVG task from `setup:ci`, not `setup` entirely. In that case, I think my question is actually moot because the situation I described shouldn't occur if someone is following the install directions. 🤦

Yeah the same install docs should still be valid

File: docs/getting-started/usage.html [hunk 1]
```diff
@@ -27,7 +27,7 @@
 </section>
 <section class="d-stack16">
   {% header "h2", "Components" %}
-  {% paragraph %}There are two methods to implement Dialtone components: Vue (recommended) and CSS. Vue is the preferred method as it's more robust and readily accessible out-of-the-box. <a href="https://handset.site/" class="d-link" target="_blank">Get started with Vue components.</a>{% endparagraph %}
+  {% paragraph %}There are two methods to implement Dialtone components: Vue (recommended) and CSS. Vue is the preferred method as it's more robust and readily accessible out-of-the-box. <a href="https://vue.dialpad.design/" class="d-link" target="_blank">Get started with Vue components.</a>{% endparagraph %}
```
Comment (braddialpad) [inline docs/getting-started/usage.html:30]: found this sneaky boi while updating the docs

**Comment by braddialpad** [comment]
> Personally I think it may be best to include the svg as part of the build. Guessing this was set up as an independent script originally for performance reasons. I'll do some build time comparisons so we can make a decision.

Okay so after testing this the build time of `npm run svg` is completely negligable (like 1 second), so I've just included it as part of the default gulp build. Makes things MUCH simpler. Updated docs/ci/netlify as well.

**Review by dbecherdialpad — APPROVED** [review]

File: docs/getting-started/usage.html (hunk 1 above)
Comment (ryandialpad) [reply docs/getting-started/usage.html:30]: Good find!

**Review by ryandialpad — APPROVED** [review]

**Pull request merged**